tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Install development headers
```

Comment:
If we only install the two Python dependencies above, these dependencies aren't necessary.